client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/components/GetHelpPage.tsx

```diff
@@ -0,0 +1,304 @@
+              : 'message-student'
+          }`}
+          onClick={() => focusEditorOnFeedbackLine(message.linenum)}
+        >
+          <ListItemText
+            className={`get-help-page-listitemtext ${
+              message.sender === 'Codaveri'
+                ? 'message-codaveri'
+                : 'message-student'
+            }`}
+            primary={
+              message.isBold ? (
+                <Typography fontWeight="bold">{message.text}</Typography>
+              ) : (
+                message.text
+              )
+            }
+            secondary={
+              message.timestamp
+                ? moment(message.timestamp).format(SHORT_DATE_TIME_FORMAT)
+                : ''
+            }
+          />
+        </ListItem>
+      ))}
+    </List>
+  );
+};
+
+const SuggestionButtons: FC<{
+  loading: boolean;
+  suggestions: string[];
+  handleSendMessage: (message: string) => void;
+}> = ({ loading, suggestions, handleSendMessage }) => (
+  <div className="get-help-page-box-center">
+    {suggestions.map((suggestion, index) => (
+      <Button
+        key={index}
+        disabled={loading}
+        onClick={() => handleSendMessage(suggestion)}
+        sx={{ mx: 1 }}
+        variant="outlined"
+      >
+        {suggestion}
+      </Button>
+    ))}
+  </div>
+);
+
+const InputField: FC<{
+  loading: boolean;
+  input: string;
+  setInput: React.Dispatch<React.SetStateAction<string>>;
+  handleKeyDown: (e: React.KeyboardEvent<HTMLInputElement>) => void;
+  handleSendMessage: (message: string) => void;
+}> = ({ loading, input, setInput, handleKeyDown, handleSendMessage }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="get-help-page-box-full-width">
+      <TextField
+        autoFocus
+        className="px-4 py-3"
+        disabled={loading}
+        fullWidth
+        label={t(translations.typeYourMessage)}
+        onChange={(e): void => setInput(e.target.value)}
+        onKeyDown={handleKeyDown}
+        size="small"
+        trims
+        value={input}
+        variant="filled"
+      />
+      <IconButton
+        className="get-help-page-iconbutton"
+        disabled={loading || input.trim() === ''}
+        onClick={() => handleSendMessage(input)}
+      >
+        {loading ? <CircularProgress size={24} /> : <Send />}
+      </IconButton>
+    </div>
+  );
+};
+
+const InputArea: FC<{
+  loading: boolean;
+  suggestions: string[];
+  questionId: number;
+  answerId: number | undefined;
+  submissionId: string | null;
+  questionIndex: number;
+}> = ({
+  loading,
+  suggestions,
+  questionId,
+  answerId,
+  submissionId,
+  questionIndex,
+}) => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const [input, setInput] = useState<string>('');
+
+  const translatedSuggestions = suggestions.map((suggestion, index) =>
+    t({
+      id: `course.assessment.submission.liveFeedback.${toCamelCase(suggestion)}`,
+      defaultMessage: suggestion,
```

Comment:
Don't do this text to camelCase, use `defineMessages` instead for `suggestions`.